app/controllers/turfs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TurfsController < ApplicationController
+  def create
+    unless current_user
+      flash[:notice] = "You must be signed in to do that."
+      render "/"
+      return
+    end
+    params["lists"].each_value do |list|
+      @list = List.new
+      @list.name = list["name"]
+      if @list.save
+        list["voter_ids"].each do |voter_id|
```

Comment:
consider creating a plain old ruby object using unit tests to manage this operation. Taking it out of the controller makes your behaviors easier to test, and keeps your controller skinny.